RNA-seq/01-qc_trim_quant.md

```diff
@@ -0,0 +1,244 @@
+
+The [documentation for individual modules/analyses](https://www.bioinformatics.babraham.ac.uk/projects/fastqc/Help/3%20Analysis%20Modules/) in FastQC is a great resource!
+
+To save time, our script only runs one FASTQ file for SRR585570 with the following command:
+
+```bash
+# In the interest of time, we'll run one of the fastq files through FastQC
+fastqc data/fastq/gastric_cancer/SRR585570/SRR585570_1.fastq.gz \
+	-o QC/gastric_cancer/fastqc_output
+```
+
+#### `-o`
+
+The `-o` flag allows us to specify where the output of FastQC is saved. 
+Note that this is saved in a different place than the raw data files and in a directory specifically for quality control information.
+
+For comparison, we've prepared a FastQC report for one of the sets of reads for another sample in the experiment.
+It can be found at `QC/gastric_cancer/fastqc_output/SRR585574_1_fastqc.html`.
+
+**Let's look at the results together.**
+
+### Preprocessing with fastp
+
+We use [fastp](https://github.com/OpenGene/fastp) to preprocess the FASTQ files ([Chen et al. _Bioinformatics._ 2018.](https://doi.org/10.1093/bioinformatics/bty560)). 
+Note that fastp has quality control functionality and many different options for preprocessing (see [all options on Github](https://github.com/OpenGene/fastp/blob/master/README.md#all-options)), most of which we will cover.
+Here, we focus on adapter trimming, quality filtering, and length filtering.
+
+Below, we discuss the two commands we used in the script.
+
+**Create a directory**
+
+```bash
+# Create a directory to hold the JSON and HTML output from Fastp
+mkdir QC/gastric_cancer/fastp_output/
+```
+
+`mkdir` allows us to create a new folder in the `QC/gastric_cancer/` directory specifically to hold the _report_ information from fastp.
+As we'll cover below, fastp essentially has two kinds of output: filtered FASTQ files (data) and reports (quality control).
+
+**fastp**
+
+```bash
+# Run the adapter and quality trimming step -- also produces QC report
+fastp -i data/fastq/gastric_cancer/SRR585570/SRR585570_1.fastq.gz \
+	-I data/fastq/gastric_cancer/SRR585570/SRR585570_2.fastq.gz \
+	-o data/fastq/gastric_cancer/SRR585570/SRR585570_fastp_1.fastq.gz \
+	-O data/fastq/gastric_cancer/SRR585570/SRR585570_fastp_2.fastq.gz \
+	--qualified_quality_phred 15 \
+	--length_required 20 \
+	--report_title "SRR585570" \
+	--json QC/gastric_cancer/fastp_output/SRR585570_fastp.json \
+	--html QC/gastric_cancer/fastp_output/SRR585570_fastp.html
+```
+
+Below, we'll walk through the arguments/options we used to run `fastp`.
+By default, fastp performs adapter trimming, which you can read more about [here](https://github.com/OpenGene/fastp#adapters). 
+For paired-end data like SRR585570, adapters can be detected automatically without specifying an adapter sequence.
+
+#### Input: `-i` and `-I`
+
+These arguments specify the read1 input and read2 input, respectively.
```

Comment:
```suggestion
These arguments specify the read1 input and read2 input, respectively (remember that paired-end sequencing contains reads that are forward and backward). 
```
This is probably not the exact right wording, but people may have forgotten at this point that this is paired-end and that's what read1 and read2 pertain to.